src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(BallsCollors.values().length);
```

Comment:
The enum values() method creates a new array every time it is called, which can be inefficient if called multiple times. Store the result of values() in a static final field to avoid this overhead.